### DISCUSSION TITLE
Memory Issue when running DGL baseline model with Nvidia Tesla V100, 120GB RAM

### DISCUSSION THREAD

**Discussion by luishengjie** [snap-stanford/ogb#196] [MAG240M-LSC]
When running the DGL baseline model `https://github.com/dmlc/dgl/blob/master/examples/pytorch/ogb_lsc/MAG240M/train.py`, I faced the following error `OSError: [Errno 12] Cannot allocate memory`. 
Machine Specification:
- Nvidia Tesla V100
- 120GB RAM

Is 120GB RAM sufficient to run the DGL baseline model?

Best Regards
Sheng Jie

**Comment by weihua916** [comment]
Please ask on DGL's Github: https://github.com/dmlc/dgl They are maintaining the code.
